### ISSUE TITLE
Error on PED file from goleft_indexcov

### ISSUE THREAD

**Issue by fgvieira** [ewels/MultiQC#1265]
**Description of bug:**

When running `multiqc` with the PED file from `goleft indexcov`, I get the error below.
@brentp, maybe can you take a look.

**MultiQC Error log:**

```
$ multiqc -f abcd_CL1005827_L01_R1-indexcov.ped -o OUT
[INFO   ]         multiqc : This is MultiQC v1.9
[INFO   ]         multiqc : Template    : default
[INFO   ]         multiqc : Searching   : work/qc/abcd_CL1005827_L01_R1/coverage/indexcov/abcd_CL1005827_L01_R1-indexcov.ped
[INFO   ]           bcbio : Found 0 reports
[INFO   ] goleft_indexcov : Found 1 samples
[ERROR  ]         multiqc : Oops! The 'goleft_indexcov' MultiQC module broke... 
  Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues 
  If possible, please include a log file that triggers the error - the last file found was:
    work/qc/abcd_CL1005827_L01_R1/coverage/indexcov/abcd_CL1005827_L01_R1-indexcov.ped
============================================================
Module goleft_indexcov raised an exception: Traceback (most recent call last):
  File "anaconda/lib/python3.6/site-packages/multiqc/plots/linegraph.py", line 173, in plot
    return get_template_mod().linegraph(plotdata, pconfig)
AttributeError: module 'multiqc.templates.default' has no attribute 'linegraph'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "anaconda/lib/python3.6/site-packages/multiqc/multiqc.py", line 569, in run
    output = mod()
  File "anaconda/lib/python3.6/site-packages/multiqc/modules/goleft_indexcov/goleft_indexcov.py", line 47, in __init__
    self.roc_plot()
  File "anaconda/lib/python3.6/site-packages/multiqc/modules/goleft_indexcov/goleft_indexcov.py", line 117, in roc_plot
    plot = linegraph.plot([self.roc_plot_data[c] for c in chroms], pconfig)
  File "anaconda/lib/python3.6/site-packages/multiqc/plots/linegraph.py", line 175, in plot
    if config.plots_force_flat or (not config.plots_force_interactive and len(plotdata[0]) > config.plots_flat_numseries):
IndexError: list index out of range
============================================================
[INFO   ]         multiqc : Compressing plot data
[WARNING]         multiqc : Deleting    : OUT/multiqc_report.html   (-f was specified)
[WARNING]         multiqc : Deleting    : OUT/multiqc_data   (-f was specified)
[INFO   ]         multiqc : Report      : OUT/multiqc_report.html
[INFO   ]         multiqc : Data        : OUT/multiqc_data
[INFO   ]         multiqc : MultiQC complete

```

**File that triggers the error:**

[abcd_CL1005827_L01_R1-indexcov.ped.txt](https://github.com/ewels/MultiQC/files/5027309/abcd_CL1005827_L01_R1-indexcov.ped.txt)

**MultiQC run details (please complete the following):**
* Command used to run MultiQC: `multiqc -f abcd_CL1005827_L01_R1-indexcov.ped -o OUT`
* MultiQC Version: `MultiQC v1.9`
* Operating System: `Ubuntu 18.04`
* Python Version: `Python 3.6.11`
* Method of MultiQC installation: `conda`

**Additional context**

**Comment by WimSpee** [comment]
@ewels  I get the same error when using multiq as wrapped in bcbio 1.2.4. 
 MultiQC does create the output html report and data dir, but has exit status 1, which causes the bcbio analysis to crash. 

MultiQC does continue after the error, finishing with this line on standard out.  But as mentioned bcbio does not continue because of the exit status 1. 

Can you please have a look. Thank you. 

```
[2020-10-02 12:26:54] multiqc                                            [INFO   ]  MultiQC complete
```

Error:
```
[ERROR  ]         multiqc : Oops! The 'goleft_indexcov' MultiQC module broke...
  Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues
  If possible, please include a log file that triggers the error - the last file found was:
    /run/Projects/DEV-1829/DA_1501/transaction/2_main_joint_analysis_dir/transaction/work/qc/DA_1474_04/coverage/indexcov/DA_1474_04-indexcov.ped
============================================================
Module goleft_indexcov raised an exception: Traceback (most recent call last):
  File "/prod/Tools/bcbio/1.2.4/anaconda/lib/python3.6/site-packages/multiqc/plots/linegraph.py", line 173, in plot
    return get_template_mod().linegraph(plotdata, pconfig)
AttributeError: module 'multiqc.templates.default' has no attribute 'linegraph'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/prod/Tools/bcbio/1.2.4/anaconda/lib/python3.6/site-packages/multiqc/multiqc.py", line 569, in run
    output = mod()
  File "/prod/Tools/bcbio/1.2.4/anaconda/lib/python3.6/site-packages/multiqc/modules/goleft_indexcov/goleft_indexcov.py", line 47, in __init__
    self.roc_plot()
  File "/prod/Tools/bcbio/1.2.4/anaconda/lib/python3.6/site-packages/multiqc/modules/goleft_indexcov/goleft_indexcov.py", line 117, in roc_plot
    plot = linegraph.plot([self.roc_plot_data[c] for c in chroms], pconfig)
  File "/prod/Tools/bcbio/1.2.4/anaconda/lib/python3.6/site-packages/multiqc/plots/linegraph.py", line 175, in plot
    if config.plots_force_flat or (not config.plots_force_interactive and len(plotdata[0]) > config.plots_flat_numseries):
IndexError: list index out of range
```

**Comment by ewels** [comment]
Should be fixed in #1310 and in the development code very shortly when that is merged 👍🏻 

Thanks @roryk!
